src/main/java/com/thealgorithms/datastructures/trees/Maximum depth of the bt.java

```diff
@@ -0,0 +1,41 @@
+package com.thealgorithms.datastructures.trees;
+import java.util.*;
+/*
+Creating class Node for tree which contain data part,left and right of that tree
+*/
+class Node{
+  int data; // data part
+  Node left; // left part of data which contains address of left-nodes
+  Node right; // right part of data which conatins address of right-nodes
+  Node(int val){  // constructor which initialize the values of data
+    data=val;
+    left=null; // initialize left address as NULL
+    right=null; // initialize right address as NULL
+  }
+}
+public class Main{ // Main class
```

Comment:
The name of the class should be the same as the name of the file. The name of the file must be is PascalCase.